user-guide/Basic_Functionality/Visio/tutorials/Using_visio_to_control_dashboard_feed_components.md

```diff
@@ -0,0 +1,239 @@
+
+> [!NOTE]
+> The data being interacted with in the example below originates from elements using the *Skyline Generic Virtual Connector* protocol. However, this process will work for elements using any protocol that features table-based parameters.
+
+## Step 1: Create a new Virtual Connector element and add numerical data to the 'Numerical Master Table' on the element's 'Table' data page
+
+1. Open DataMiner Cube.
+
+1. Create a new element using the *Skyline Generic Virtual Connector* protocol.
+
+   1. Open the Surveyor.
+
+   1. Right-click a view (e.g. the root view), and select *New* > *Element*.
+
+   1. Enter an element name, select the DMA, the protocol (i.e. *Skyline Generic Virtual Connector*) and the protocol version (e.g. 1.0.0.3), and click *Create*.
+
+1. Go to the *DATA* > *Table* page of the newly created element, which contains the *Numeric Master Table*.
+
+1. Right-click below the table's column headers, and click *Add* to add a new row to the table.
+
+   ![Add Row Numeric Table](~/user-guide/images/VisioURLFeed_AddRowNumericTable.png)
+
+1. Enter the row's primary key [IDX], enter random numbers into the first three columns, and click *OK*.
+
+   > [!IMPORTANT]
+   > Do not use any special characters for the primary key, as they will interfere with the URL linking later in this tutorial.
+
+   ![Table input](~/user-guide/images/VisioURLFeed_AddRowNumericTableInput.png)
+
+1. Add at least one more row to the table.
+
+   ![Table rows](~/user-guide/images/VisioURLFeed_NumericTableRows.png)
+
+## Step 2: Create a new dashboard and add a 'Parameter feed' component and a 'State' component to display the numerical data from the Virtual Connector
+
+1. Open the *Dashboards* app.
+
+1. [Create a new dashboard](xref:Creating_a_completely_new_dashboard).
+
+1. If you are not yet in edit mode, then click *Start editing*.
+
+1. Add a [Parameter feed component](xref:DashboardParameterFeed).
+
+1. From the *Data* pane on the right, drag the *ELEMENTS* header onto the *Parameter feed* component to be used as data.
+
+1. From the *Data* pane on the right, add the necessary parameters:
+
+   1. Expand the *PARAMETERS* section.
+
+   1. Set *From* to "Protocol".
+
+   1. Set *Protocol* to "Skyline Generic Virtual Connector".
+
+   1. Use the filter to find the three numeric table parameters that you entered numbers for.
+
+   1. Drag each of these parameters onto the *Parameter feed* component to be used as data.
+
+   ![Parameters](~/user-guide/images/VisioURLFeed_ParameterDataForFeed.png)
+
+1. From the *Data* pane on the right, add a protocol filter:
+
+   1. Expand the *PROTOCOLS* section.
+
+   1. Search for the *Skyline Generic Virtual Connector*.
+
+   1. Drag the protocol onto the *Parameter feed* component to be used as a filter.
+
+   ![Protocols](~/user-guide/images/VisioURLFeed_ProtocolDataForFeed.png)
```

Comment:
Could you add a better screenshot here that really shows how you add this as a filter?

Reply:
Done